lighthouse-core/audits/non-composited-animations.js

```diff
@@ -27,6 +27,16 @@ const UIStrings = {
     =1 {Unsupported CSS Property: {properties}}
     other {Unsupported CSS Properties: {properties}}
   }`,
+  /** Name of a compositor failure reason where the transform being animated depends on the box size. */
+  transformDependsBoxSize: 'Transform related property depends on box size',
+  /** Name of a compositor failure reason where the filter being animated may move pixels. */
+  filterMayMovePixels: 'Filter related property may move pixels',
+  /** Name of a compositor failure reason where the effect has a composite mode which is not replace. */
+  nonReplaceCompositeMode: 'Effect has composite mode other than "replace"',
+  /** Name of a compositor failure reason where the element already has an incompatible animation. */
+  incompatibleAnimations: 'Target has an incompatible animation',
+  /** Name of a compositor failure reason where the effect has unsupported timing parameters. */
+  unsupportedTimingParameters: 'Effect has unsupported timing parameters',
```

Comment:
this feels like one that will definitely need a document to explain, what about them is invalid? what are the valid options? is any of that possibly explainable in this message or should we leave it here and always point to docs?

Reply:
[doc](https://docs.google.com/document/d/1XKcJP2CKmNKfOcDsVvliAQ-e1H9C1nf2H-pzTdyafAA/edit#)

I don't think the reason can explain the valid options because there are a few failure cases.

Reply:
gotcha, yeah we'll want to get some web.dev docs up for this one otherwise I'm not sure anyone is going to be able to fix it :)